osu.Game/Screens/Play/BeatmapMetadataDisplay.cs

```diff
@@ -225,10 +226,10 @@ public MetadataLineLabel(string text)
 
         private class MetadataLineInfo : OsuSpriteText
         {
-            public MetadataLineInfo(string text)
+            public MetadataLineInfo(LocalisableString text)
             {
                 Margin = new MarginPadding { Left = 5 };
-                Text = string.IsNullOrEmpty(text) ? @"-" : text;
+                Text = string.IsNullOrEmpty(text.ToString()) ? @"-" : text;
```

Comment:
This isn't being used anywhere, and the null check is weird, so I'll revert back.